### DISCUSSION TITLE
Categories are not applying / Refreshing

### DISCUSSION THREAD

**Discussion by NanaimoaAA** [code4recovery/12-step-meeting-list#910] [Q&A] [answered]
Hello, 

I am having an issue with categories not applying without a refresh. For example if I select a different city in the region drop down nothing happens. I can see the url update, and if I hit refresh it will apply.  I have disabled all plugins and re-installed and the same issue is present.

This is my first time posting here, so any support would be great. And let me know if I did it correctly. 

[](https://www.dropbox.com/s/80bg0a7gebpvqu6/MIIS%20Category%20problem.mp4?dl=0)

Thank you, 

https://user-images.githubusercontent.com/113145640/189239679-d8fd9ff6-6e5d-4032-8628-920e0752dedb.mp4

**Comment by kiyote33** [comment]
Hi NanaimoAA, any selection you make in any of the category dropdowns should be applied and results viewed immediately. Can you tell me a little about your environment (WordPress version, theme, installed plugins, browsers used?)?

**Reply by NanaimoaAA** [reply]
Hello, thank you for the quick reply.

WordPress version 6.0.2 
theme: Astra+elementor
installed plugins: only meetings is enabled in the example
browsers used: Chrome

**Reply by kiyote33** [reply]
Hmmm. I'm running Astra+Elementor on a site I support, so we can rule out the theme being a problem. I think Josh might be onto something with the host caching issue. I'll wait to hear back on your investigation on that...

**Comment by joshreisner** [comment, marked as answer]
its great you disabled plugins. do you happen to have website caching enabled, maybe at the hosting level? we heard of a similar situation recently with godaddy wordpress caching.

**Reply by NanaimoaAA** [reply]
thank you for the reply. The hosting is done through siteground. I will poke around and see what I can find.

**Reply by NanaimoaAA** [reply]
Through the power of google, I followed the steps below on disabling caching as well clearing what is currently there. Issue persists. 
[](https://www.siteground.com/kb/disable-dynamic-caching-website/)

**Reply by NanaimoaAA** [reply]
I also connected with Siteground support about this. I will post the chat here. 

07:18 PM (Siteground)
I have stopped the dynamic cache, and cleared all cache, from here you will need to check the plugin and how it works, and if it does not, it will be best to check with the vendors of the plugin

07:18 PM
okay thank you. It is still happening,  I will connect with the vendors. 

07:20 PM (Siteground)
This will be for the best since I cannot see anything else that would be causing this .

**Reply by joshreisner** [reply]
Ok, thanks. Interesting that there was a cache enabled at the hosting level.

Sounds like javascript AJAX requests are not working for some reason. The next step would be to check the javascript console to see what those are [instructions](https://developer.chrome.com/docs/devtools/open/)

I'd do it for you but looks like your staging server is pw protected

**Reply by NanaimoaAA** [reply]
Good morning Josh, you can access http://staging17.nanaimoaa.org/ with un:guest pw:officeguest. The live site is nanaimoaa.org, but I figured I should be testing on a staging site with all the plugins turned off and then apply to the live site.

**Reply by joshreisner** [reply]
there does appear to be some invalid HTML in the "meetings top" widget area, which could be affecting things. could you clear that out?

i hope that fixes it! if not, can you recall anything that might have changed around the time it stopped working?

**Reply by NanaimoaAA** [reply]
Probably stopped working around the time that I edited the meetings top widget area :P . That fixed it! Thank you soooooo much.
